pkg/api/v1/helper/helpers_test.go

```diff
@@ -499,3 +500,155 @@ func TestGetAffinityFromPodAnnotations(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPodQOS(t *testing.T) {
```

Comment:
I'd like to have a test (I think this one), that does a conversion to internal and runs the internal version of the function to make sure that it agrees with the external version of the function.